### DISCUSSION TITLE
Quasar text area for chat input

### DISCUSSION THREAD

**Discussion by tianhuat** [quasarframework/quasar#11343] [General - Components / Directives / etc]
The following is the text area found in 
https://quasar.dev/vue-components/input#textarea

I highlighted the modification I wish to  have into different color of rectangle - I wish to make it a chat input text area.

1. red rectangle - can I disable the underlying blue line? 
1. green rectangle - can I disable the resizable option? 
1. yellow rectangle -  can I make the scroll bar smaller?

[![enter image description here][1]][1]

The ideal is the following

[![enter image description here][2]][2]

  [1]: https://i.stack.imgur.com/ABKkA.png
  [2]: https://i.stack.imgur.com/zu4ba.png

**Comment by pikil** [comment]
There are props like `borderless` and `autogrow` available in q-input for points No.1 and No.2, for point No.3 you can add a custom scrollbar via `input-class` (not supported in all browsers as of now, though).

https://quasar.dev/vue-components/input#example--autogrow
https://quasar.dev/vue-components/input#borderless
https://www.w3schools.com/howto/howto_css_custom_scrollbar.asp

Hope that helps :)
